### DISCUSSION TITLE
Development Simulator

### DISCUSSION THREAD

**Discussion by FizzyApple12** [Ralim/IronOS#1454] [Q&A] [answered]
Is there a tool which exists to let me test new firmware additions on a simulator? My Pinecil v2 is currently stuck in transit and it would be helpful to test UI and feature changes that I'm currently working on in a simulator.

**Comment by Ralim** [comment, marked as answer]
Hello, as of present is no simulator.
Not against there being one but have not had time to make one.

**Reply by FizzyApple12** [reply]
Alright, I may throw something together if I can find the time, thank you!

**Reply by discip** [reply]
That would be much appreciated!
So we would not have to shorten the life of the memory by flashing firmware to be tested over and over again.
